src/components/chat/MessageForm.js

```diff
@@ -73,45 +74,64 @@ class MessageForm extends Component {
     this.props.onSendMessage(message);
     this.clearMessage();
   }
+
   togglePicker = () => {
     this.setState((prevState) => {
       return {showEmojiPicker: !prevState.showEmojiPicker}
     })
   }
+
+  onClickPicker = (emoji, event) => {
+    this.addEmoji(emoji);
+    this.closePicker();
+  }
+
+  backgroundImageFn = (set, sheetSize) => {
+    if (set !== 'apple' || sheetSize !== 64) {
+      console.log('WARNING: using set "apple" and sheetSize 64 rather than',
+                  set, 'and', sheetSize, 'as was requested');
+    }
+    return '/' + emoji.EMOJI_APPLE_64_SHEET;
+  }
+
   closePicker = () => {
     this.setState({
       showEmojiPicker: false
     })
   }
+
   addEmoji = (emoji) => {
     this.setState((prevState) => {
       return {message: prevState.message.concat(`${emoji.colons} `)}
     })
   }
+
   render() {
     let { message, showEmojiPicker } = this.state;
 
     return (
       <div className="message-form">
         <form role="form" className="form" onSubmit={this.onSendMessage}>
-          {showEmojiPicker && <Picker 
-             emojiSize={24}
-             perLine={9}
-             skin={1}
-             set={'apple'}
-             autoFocus={false}
-             include={[]}
-             exclude={['nature', 'places', 'flags']}
-             emoji={""}
+          {showEmojiPicker && <Picker
+            emojiSize={24}
+            perLine={9}
+            skin={1}
+            set={'apple'}
+            autoFocus={false}
+            include={[]}
+            exclude={['nature', 'places', 'flags']}
+            emoji={""}
             title={"LeapChat"}
-            onClick={(emoji, event) => {this.addEmoji(emoji); this.closePicker()}}/>}
-          <div>
+            backgroundImageFn={this.backgroundImageFn}
```

Comment:
@Moe-Shoman This is the only tangible change in this area; the rest is whitespace